app/src/main/java/org/mozilla/fenix/settings/advanced/LocaleAdapter.kt

```diff
@@ -98,7 +99,7 @@ class LocaleAdapter(private val interactor: LocaleSettingsViewInteractor) :
 }
 
 class LocaleViewHolder(
-    view: View,
+    private val view: View,
```

Comment:
Instead of duplicating a reference to the view, you can use `itemView`. (ie: `itemView.context`)

Reply:
True, removed the view reference, thanks!